code/game/objects/buckling.dm

```diff
@@ -61,16 +61,16 @@
 	if(istype(M, /mob/living/carbon/slime))
 		user << "<span class='warning'>The [M] is too squishy to buckle in.</span>"
 		return
+	if (buckled_mob)
+		user << "<span class='warning'>[buckled_mob.name] is already there, unbuckle them first!.</span>"
+		return
 
 	add_fingerprint(user)
-	unbuckle_mob()
+	unbuckle_mob()//this is now just for safety, buckling someone into an occupied chair will fail, instead of removing the occupant
 
 	if(buckle_mob(M))
 		if(M == user)
-			M.visible_message(\
-				"<span class='notice'>[M.name] buckles themselves to [src].</span>",\
-				"<span class='notice'>You buckle yourself to [src].</span>",\
-				"<span class='notice'>You hear metal clanking.</span>")
+			M << "<span class='notice'>You buckle yourself to [src].</span>"

```

Comment:
This change doesn't make sense?

It makes sense that someone buckling themselves onto a chair/bed/whatever sends a visible message.